packages/main/src/plugin/telemetry/telemetry.spec.ts

```diff
@@ -172,6 +188,12 @@ test('Check propagate enablement event if configuration is updated', async () =>
   expect(hasBeenEnabled).toBeTruthy();
 });
 
+test('Enterprise configuration telemetry info is loaded upon init', async () => {
+  await telemetry.init();
+  expect(vi.mocked(defaultConfigurationMock.getTelemetryInfo)).toHaveBeenCalled();
+  expect(vi.mocked(lockedConfigurationMock.getTelemetryInfo)).toHaveBeenCalled();
```

Comment:
And finally, you don't need to use `vi.mocked()` inside `expect`, it will work with the shortcut:


```suggestion
  expect(defaultConfigurationMock.getTelemetryInfo).toHaveBeenCalled();
  expect(lockedConfigurationMock.getTelemetryInfo).toHaveBeenCalled();
```